src/python/zquantum/qcbm/ansatz_test.py

```diff
@@ -284,58 +342,70 @@ def test_ansatz_circuit_six_layers(self):
         circuit = ansatz.get_executable_circuit(params)
 
         # Then
-        self.assertEqual(circuit, expected_circuit)
+        assert circuit == expected_circuit
 
-    def test_ansatz_circuit_seven_layers(self):
+    def test_ansatz_circuit_sevenumber_of_layers(self, number_of_qubits, topology):
```

Comment:
This function name is wrong. Not sure what you meant it to be, but there's a typo